index.js

```diff
@@ -482,6 +482,120 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
   }
 })
 
+app.post('/api/siren-addressed', Validator.body(['coreid', 'event', 'data']).exists(), async (request, response) => {
+  try {
+    const validationErrors = Validator.validationResult(request).formatWith(helpers.formatExpressValidationErrors)
+
+    if (validationErrors.isEmpty()) {
+      const coreId = request.body.coreid
+      const location = await db.getLocationFromParticleCoreID(coreId)
+
+      if (!location) {
+        const errorMessage = `Bad request to ${request.path}: no location matches the coreID ${coreId}`
+        helpers.logError(errorMessage)
+        // Must send 200 so as not to be throttled by Particle (ref: https://docs.particle.io/reference/device-cloud/webhooks/#limits)
+        response.status(200).json(errorMessage)
+      } else {
+        let client
+
+        try {
+          client = await db.beginTransaction()
+          if (client === null) {
+            helpers.logError(`alertSessionChangedCallback: Error starting transaction`)
```

Comment:
I think `alertSessionChangedCallback` was copied here but should be changed to be more descriptive of where this error is coming from.

Reply:
It was copied, changed to sirenAddressedCallback